scripts/build/build.sh

```diff
@@ -69,17 +75,20 @@ function build_backend_linux_amd64() {
   if [ ! -d "dist" ]; then
     mkdir dist
   fi
-  CC=${CCX64} go run build.go "${OPT}" build
+  CC=${CCX64} go run build.go ${OPT} build
```

Comment:
I'm here removing the quotes around `${OPT}` to prevent multiple flags (in `$OPT`) from being passed as a single argument.